api/src/main/java/org/pmiops/workbench/db/dao/projection/PrjUser.java

```diff
@@ -0,0 +1,98 @@
+package org.pmiops.workbench.db.dao.projection;
+
+import java.sql.Timestamp;
+
+public interface PrjUser {
```

Comment:
The `Prj` prefix is unfortunate, but luckily we only need to refer to these classes from a couple of places.

Reply:
Is there a reason to make it short like this? e.g. instead of `ProjectedUser`

Reply:
We have like 4 types now.
* db
* projection
* dto
* api model

I figure `DbUser` has a short prefix so maybe I should follow that pattern. In prractice, we could have several different projection interfaces pointing to DbUser with more or fewer fields, so we might need `ProjectedReportingUser` or something.